app/src/main/java/com/feedhenry/helloworld/HelloFragment.java

```diff
@@ -67,7 +67,7 @@ private void cloudCall(final View button, final String name, final TextView resp
                 public void success(FHResponse fhResponse) {
                     Log.d(TAG, "cloudCall - success");
                     button.setEnabled(true);
-                    response.setText(fhResponse.getJson().getString("msg"));
+                    response.setText(fhResponse.getJson().getString("text"));
```

Comment:
I'm not sure about this, it runs against helloworld-cloud and there is still "msg".
https://github.com/feedhenry-templates/helloworld-cloud/blob/master/lib/hello.js#L17

Reply:
Yeah, app crashes because of this

Reply:
Hrm, I did not touched on it, but I have commited a fix